HST/hst_helper/query_utils.py

```
@@ -256,11 +291,69 @@ def download_files(table, dir, logger=None, testing=False):
+                    # if filename_fn and os.path.isfile(final_path):
+                    #     logger.info(
+                    #         f'Skipping download for {product_fname}: file already exists'
+                    #     )
+                    #     continue
+
+                    fname = (filename_fn(row)
+                             if filename_fn
+                             else product_fname)
+                    local_path = os.path.join(product_dir, fname)
+
+                    status, msg, url = Observations.download_file(
```

Ruff (0.15.17)

[warning] Unpacked variable `url` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)